extract.py

```diff
@@ -4,12 +4,29 @@
 import pandas as pd
 
 BASE_URL = 'https://data-eng-plants-api.herokuapp.com/plants/'
+NO_OF_PLANTS = 51
```

Comment:
in the future we could maybe grab this from the base URL of the API in case it changes, but fine for now